### ISSUE TITLE
Have ARIA status message to make Elk be fully SR-proof

### ISSUE THREAD

**Issue by edimitchel** [elk-zone/elk#369]
On any action made on Elk, today, there is not status shared for screen readers.
Only things has been done for making UI readable, but nothing (on what I see) had been done for describing status or result of action.

ARIA provides roles for sharing those information: 
I found a great resource about and could be great to be implemented:
https://a11y-guidelines.orange.com/en/articles/aria-status-messages/

## [status role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role)

- [ ] status publised
- [ ] sharing new tweet available
- [ ] shortcut executed (especially in case of redirection)
 #39 : for page navigation sharing also feedback visually like a discreet alert on bottom center

## [alert role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role)

- [ ] new notification
- [ ] a new reply

List to be completed

**Comment by edimitchel** [comment]
Joaquin, what about this issue, you've completed almost all point, right?

**Comment by userquin** [comment]
yeah, but not being used :sweat_smile: , we should review Elk, you can check `components/aria` folder
